packages/strapi-design-system/src/Pagination/components.js

```diff
@@ -1,71 +1,133 @@
 import React from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
-/** TODO: make sure to use the react component when they are available in strapi-icons */
-import prevLink from './assets/previous.svg';
-import nextLink from './assets/next.svg';
+import { BackFilter, NextFilter } from '@strapi/icons';
 import { VisuallyHidden } from '../VisuallyHidden';
-import { useActivePage } from './PaginationContext';
+import { usePagination } from './PaginationContext';
 import { Text } from '../Text';
 
+const PaginationText = styled(Text)`
+  line-height: revert;
+`;
+
 // TODO: make sure to use the Link exposed by the chosen router
-const Wrapper = styled.a`
+const LinkWrapper = styled.a`
   padding: ${({ theme }) => theme.spaces[3]};
-  background: ${({ theme, active }) => (active ? theme.colors.neutral0 : theme.colors.neutral100)};
   border-radius: ${({ theme }) => theme.borderRadius};
-  color: ${({ theme, active }) => (active ? theme.colors.primary600 : theme.colors.neutral800)};
   box-shadow: ${({ active }) => (active ? `0px 1px 4px rgba(26, 26, 67, 0.1)` : undefined)};
   text-decoration: none;
   display: flex;
 `;
 
-const PaginationText = styled(Text)`
-  line-height: revert;
+const PageLinkWrapper = styled(LinkWrapper)`
+  color: ${({ theme, active }) => (active ? theme.colors.primary700 : theme.colors.neutral800)};
+  background: ${({ theme, active }) => (active ? theme.colors.neutral0 : undefined)};
+
+  &:focus {
+    ${PaginationText} {
+      font-weight: bold;
+    }
+  }
+
+  &:hover {
+    box-shadow: 0px 1px 4px rgba(26, 26, 67, 0.1);
+
+    ${PaginationText} {
+      font-weight: bold;
+    }
```

Comment:
This affects the pagination wrapper width. Is it a problem @maevalienard @rgesulfo?

![pagination](https://user-images.githubusercontent.com/7756284/117277908-9253f000-ae60-11eb-934e-fc785f20131c.gif)

Reply:
It's because we switch from regular text to a bold one. We can remove the boldness if needed

Reply:
@maevalienard to make sure I update the Figma correctly, we remove the bold weight on the hover and select states? Or just the hover? Thanks